### PR TITLE
Add more tests

### PR DESCRIPTION
**Pull request by Luni-4**
This PR fixes the last item of #1 

@lu-zero
I can't figure out the first and the second item of the issue, could you elaborate more on them, please? Thanks in advance! :)

### CODE REVIEW

**Review by lu-zero — APPROVED** [review]
This set is already good to land :)

Thank you, let me check the issue and clarify.

**Comment by lu-zero** [comment]
I clarified and dropped the timestamp check since ivf doesn't really have the problem.

File: tests/remuxer.rs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            }
+            (_, _) => {
+                panic!("The two demuxers do not get the same output");
+            }
+        }
+    }
+}
+
+#[test]
+fn remuxer() {
```
Comment (lu-zero) [inline tests/remuxer.rs:111]: Would be better if the output stays in-memory, are you willing to update the test?

Comment (Luni-4) [reply tests/remuxer.rs:111]: Sure! So we could solve the second item of #1

**Pull request merged**